timeline.js

```diff
@@ -1,51 +1,53 @@
-import { db } from "./dbInstance.js";
+import { db } from './dbInstance.js'
+
+let hasLoaded = false
 
 class ReaderTimeline extends HTMLElement {
-  constructor() {
-    super();
-    this.actorUrls = [
-      "https://staticpub.mauve.moe/about.jsonld",
-      "https://hypha.coop/about.jsonld",
-      "https://prueba-cola-de-moderacion-2.sutty.nl/about.jsonld",
-    ];
-    this.processedNotes = new Set(); // To keep track of notes already processed
+  connectedCallback () {
+    this.initializeDefaultFollowedActors().then(() => this.initTimeline())
   }
 
-  connectedCallback() {
-    this.initTimeline();
+  async initializeDefaultFollowedActors () {
+    const defaultActors = [
+      'https://social.dp.chanterelle.xyz/v1/@announcements@social.dp.chanterelle.xyz/',
```

Comment:
We should replace this with the prod announcements
```suggestion
      'https://social.distributed.press/v1/@announcements@social.distributed.press/',
```